docs/source/installation/troubleshooting.rst

```diff
@@ -193,31 +195,29 @@ Follow the steps in `this StackExchange answer <https://superuser.com/questions/
 --------------------------------
 
 If ``choco install manimce`` failed,
-it is likely being caused by Python not being added to your ``PATH`` variable properly.
+it is because Python was not added to your ``PATH`` variable properly.
```

Comment:
This does change the meaning somewhat, and I'm not sure the statement is still true after the change. This should be updated to reflect that if ``choco install manimce`` it does not **always** have to be this particular problem.

Reply:
I think this part of troubleshooting can be removed. In the recent version the installation script searches the windows registry for python and doesn't depend on having it in the PATH variable.

Reply:
I'll go ahead and remove this line if it's superfluous, then.

Reply:
Hm... a large section (lines 188t to least 213) refers to the PATH variable... which should I remove and which should I keep? For right now, I'll just update it to say likely.